LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
```

Comment:
+1, I was thinking it would just default to evm by default if no $1 is provided.

But that was just a thought.

This all looks good to me :shipit: